config.txt

```diff
@@ -5,6 +5,7 @@
 $class: 'CCUCMScm'
 $class: 'CVSSCM'
 $class: 'CvsProjectset'
+scmGit
```

Comment:
FWIW I didn't think symbols were supposed to include their implementation in their name, is their a reason it's called scmGit instead of `git`?

Reply:
The `git` symbol is already used by the `git` step that is implemented in the git plugin.  Since I couldn't name it `git`, I had to find some other name.  I started with `gitSCM` but it was noted that uses capitalization that is inconsistent with the usual symbol capitalization like `resolveScm`.  I didn't like `gitScm`, so I used `scmGit`.

The history of that choice is in the git plugin pull request https://github.com/jenkinsci/git-plugin/pull/1373 and in the hundreds of individual commits that I eventually squashed into a very few commits.

Reply:
Symbol's are unique to their extension point though.

The base class is https://github.com/jenkinsci/jenkins/blob/850a5d85200d217363a4c7806be8217e1c312f21/core/src/main/java/hudson/scm/SCM.java

I assume there was an issue though?

Reply:
Yes, there was an observed issue with assigning `git` as the symbol for the describable that now uses the `scmGit` symbol.  

The experiment attempted to use `git` as preferred symbol instead of `gitSCM` in commit [cb5e3f42f](https://github.com/MarkEWaite/git-plugin/commit/cb5e3f42f2843ea1fca91b243800290d16fc9e0a).  That commit was reverted in [a7bcd86](https://github.com/MarkEWaite/git-plugin/commit/a7bcd86387e01806d538822fa7a7edf88fc17944) because it caused the Pipeline Syntax Snippet Generator to again display `$class: GitSCM` instead of displaying `git`.  The final removal was in [04d2db6196](https://github.com/MarkEWaite/git-plugin/commit/04d2db6196a31dcbf77d48b4ea3388c619f86dcb).

Reply:
Right, I tracked it down to https://github.com/jenkinsci/workflow-cps-plugin/blob/9b36ad99e1ccd380d8e2e1701be862ea0761e295/plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java#L274-L277